docs/src/main/sphinx/release/release-463.md

```diff
@@ -0,0 +1,27 @@
+# Release 463 (23 Oct 2024)
+
+## General
+
+* Add {func}`timezone` functions to extract the timezone identifier from a
+  timestamp or time. ({issue}`20893`)
+* Add table functions to the `SHOW FUNCTIONS` output. ({issue}`12550`)
+* Disallow the window framing clause for {func}`ntile`, {func}`rank`,
+  {func}`dense_rank`, {func}`percent_rank`,  {func}`cume_dist`, and
+  {func}`row_number`. ({issue}`23742`)
+
+## ClickHouse connector
+
+* Improve performance for queries with `IS NULL` expressions. ({issue}`23459`)
+
+## Delta Lake connector
+
+* Add support for writing change data feed when [deletion vector](https://docs.delta.io/latest/delta-deletion-vectors.html) 
+  is enabled. ({issue}`23620`)
+
+## Iceberg connector
+
+* Add support for nested namespaces with the REST catalog. ({issue}`22916`)
+* Add support for configuring the maximum number of rows per row-group in the
+  ORC writer with the `orc_writer_max_row_group_rows` catalog session property. ({issue}`23722`)
```

Comment:
Added the name of the property like in earlier release notes @raunaqmorarka  ... please confirm that it is a catalog session property.

Also we dont want docs for any of them?

And they are Iceberg only?

Reply:
it is a catalog session property in iceberg.
i would skip documentation for now, it's very technical detail